slasher/beaconclient/receivers.go

```diff
@@ -33,6 +39,13 @@ func (bs *Service) receiveBlocks(ctx context.Context) {
 			log.WithError(ctx.Err()).Error("Context canceled - shutting down blocks receiver")
 			return
 		}
+		if err != nil && strings.Contains(strings.ToLower(err.Error()), strings.ToLower(context.Canceled.Error())) {
```

Comment:
I think you can do
if ctx.Err() == context.Canceled {

Reply:
i tried it and it doesn't work as the error is rpc error and context cancelled is and inner error with lower letters

Reply:
fixed.
thanks